src/content/nft/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Non-fungible tokens (NFT)
+description: An overview of NFTs on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth2/core.png
+summaryPoints:
+  [
+    "A way to turn anything unique into an Ethereum-based asset.",
+    "NFTs are giving more power to content creators than ever before.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+NFTs are currently taking the digital art and collectibles world by storm. Digital artists are seeing their lives change thanks to huge sales to a new crypto-audience. And celebrities are joining in as they spot a new opportunity to connect with fans.
+
+Here's what you need to know about NFTs.
+
+## What's an NFT?
+
+NFTs are tokens that we can use to represent a record of ownership. They let us tokenise art, collectibles, even purely physical items like designer trainers/sneakers. They can only have one owner at a time and they're protected by the Ethereum blockchain – no one can modify the record of ownership or copy/paste a new NFT into existence.
+
+_If Andy Warhol had been born in the late 90s, he probably would have minted Campbell's Soup as an NFT. It's only a matter of time before Kanye puts a run of Yeezys on Ethereum. And one day owning your car and your rental agreement might be managed with NFTs._
+
+### Examples
+
+THe NFT world is relatively new. In theory, the scope for NFTs is anything that is unique that needs provable ownership. Here are some examples of NFTs that exist today, to help you get the idea:
+
+- A unique digital artwork.
+- #23/#100 of a limited edition run of designer sneakers.
+- An in-game item.
+- An essay.
+- A digital collectible like a trading card.
+
+## How do NFTs work?
+
+NFTs have some special properties:
+
+- each token is unique.
+- they're not interchangeable with another NFT in the same run.
+- each token has an owner and this information is easily verifiable.
+- you can't own a % of an NFT token in the same way you can own 0.0001 ETH.
```

Comment:
Perhaps "indivisible" is a simple way to explain this?

Reply:
Yeah, I like the point but the `%` threw me off a little

Reply:
apparently there are projects out there that let you own a percentage of an NFT so I'll remove this point...